BentoKit/BentoKit/Components/MultilineTextInput.swift

```diff
@@ -29,40 +40,73 @@ extension Component.MultilineTextInput {
+        fileprivate let placeholderLabel = UILabel().with {
+            $0.translatesAutoresizingMaskIntoConstraints = false
+        }
+        fileprivate let contentView = UIView()
+        fileprivate let sendButton = UIButton(type: .system).with {
+            $0.translatesAutoresizingMaskIntoConstraints = false
+            $0.setContentHuggingPriority(.required, for: .vertical)
+            $0.setContentHuggingPriority(.required, for: .horizontal)
+            $0.setContentCompressionResistancePriority(.required, for: .horizontal)
+            $0.setContentCompressionResistancePriority(.required, for: .vertical)
+            $0.widthAnchor.constraint(greaterThanOrEqualToConstant: 30).activated()
+            $0.heightAnchor.constraint(greaterThanOrEqualToConstant: 30).activated()
+        }
+
+        var didFinishEditing: Optional<(String) -> Void> = nil
+        var didChangeText: Optional<(String) -> Void> = nil
 
         private var lastKnownContentHeight: CGFloat = 0.0
 
         public override init(frame: CGRect) {
             super.init(frame: frame)
 
             textView.delegate = self
-            textView.add(to: self).pinEdges(to: layoutMarginsGuide)
+            sendButton.addTarget(
+                self,
+                action: #selector(sendButtonPressed),
+                for: .primaryActionTriggered
+            )
 
-            addSubview(placeholderLabel)
-            placeholderLabel.translatesAutoresizingMaskIntoConstraints = false
+            contentView.add(to: self).pinEdges(to: layoutMarginsGuide)
+
+            stack(.horizontal, spacing: 4, alignment: .bottom)(
+                textView,
+                sendButton
+            )
+            .add(to: contentView)
+            .pinEdges(to: contentView.layoutMarginsGuide)
+
+
+            contentView.addSubview(placeholderLabel)
 
             // Pin the placeholder label to the layout margins guide. The bottom
             // edge should have lower priority than 1000 so that the UITextView
             // can take precedence when the content height is being determined.
             NSLayoutConstraint.activate([
-                layoutMarginsGuide.topAnchor.constraint(equalTo: placeholderLabel.topAnchor),
-                layoutMarginsGuide.leadingAnchor.constraint(equalTo: placeholderLabel.leadingAnchor),
-                layoutMarginsGuide.trailingAnchor.constraint(equalTo: placeholderLabel.trailingAnchor),
-                layoutMarginsGuide.bottomAnchor.constraint(equalTo: placeholderLabel.bottomAnchor)
+                contentView.layoutMarginsGuide.topAnchor.constraint(equalTo: placeholderLabel.topAnchor),
+                contentView.layoutMarginsGuide.leadingAnchor.constraint(equalTo: placeholderLabel.leadingAnchor),
+                contentView.layoutMarginsGuide.trailingAnchor.constraint(equalTo: placeholderLabel.trailingAnchor),
+                contentView.layoutMarginsGuide.bottomAnchor.constraint(equalTo: placeholderLabel.bottomAnchor)
```

Comment:
[nit]
```swift
placeholderLabel.pinToEdges(contentView.layoutMargins)
```

Reply:
We can't use `pinToEdges` here because the bottom constraint has a lower priority (`.defaultHigh`)

Reply:
```swift
 func pinEdges(to layoutGuide: UILayoutGuide,
                  insets: UIEdgeInsets = .zero,
                  topPriority: UILayoutPriority = .required,
                  bottomPriority: UILayoutPriority = .required,
                  leadingPrioryty: UILayoutPriority = .required,
                  trailingPriority: UILayoutPriority = .required)
```
:)